Makefile

```diff
@@ -43,7 +43,7 @@ prepare-build-image: build
 	cp broker build/broker
 
 build-image: prepare-build-image
-	docker build -f ${BUILD_DIR}/Dockerfile-src ${BUILD_DIR} -t ${BROKER_IMAGE}:${TAG}
+	docker build -f ${BUILD_DIR}/Dockerfile-src -t ${BROKER_IMAGE}:${TAG} ${BUILD_DIR}
```

Comment:
Yeah, no problem. Don't know why but it seems to only not work for me, my docker only allows `docker build [OPTIONS] DIR`